hub/core/dataset/dataset.py

```diff
@@ -2294,13 +2294,16 @@ def save_view(
                 - If not specified, the VDS is saved under `.queries` subdirectory of the source dataset's storage.
                 - If the user doesn't have write access to the source dataset and the source dataset is a hub cloud dataset, then the VDS is saved is saved under the user's hub account and can be accessed using `hub.load(f"hub://{username}/queries/{query_hash}")`.
             id (Optional, str): Unique id for this view. Random id will be generated if not specified.
-            optimize (bool): Whether the view should be optimized by copying the required data. Defaults to False.
+            optimize (bool): - If True, the dataset view will be optimized by copying and rechunking the required data. This is
+                necessary to achieve fast streaming speeds when training models using the dataset view. The optimization process will
+                take some time, depending on the size of the data.
+                - You can also choose to optimize the saved view later: See `hub.core.dataset.view_entry.ViewEntry.optimize`.
             num_workers (int): Number of workers to be used if `optimize` is True.
             ds_args (dict): Additional args for creating VDS when path is specified. (See documentation for `hub.dataset()`)
 
         Note:
             Specifying `path` makes the view external. External views cannot be accessed using the parent dataset's `Dataset.get_view`,
-            `Dataset.load_view`, `Dataset.delete_view` methods. They have to be loaded using `hub.load(path)`.
+            `Dataset.load_view`, `Dataset.delete_view` methods. They have to be loaded using `hub.\0load(path)`.
```

Comment:
not sure what this null character is doing here?

Reply:
otherwise hub.load gets replaced by `dataset.load()`. It's their automatic formatting smh